src/modules/dashboard/components/ActionsPage/TotalStakeWidget/TotalStakeWidget.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useEffect } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import formatNumber from 'format-number';
+
+import Heading from '~core/Heading';
+import ProgressBar from '~core/ProgressBar';
+import { useLoggedInUser } from '~data/index';
+import {
+  useStakeAmountsForMotionQuery,
+  useColonyNativeTokenQuery,
+  useTokenInfoLazyQuery,
+} from '~data/generated';
+import { Address } from '~types/index';
+
+import styles from './TotalStakeWidget.css';
+
+const displayName = 'TotalStakeWidget';
+
+type Props = {
+  tokenDecimals: number;
+  colonyAddress: Address;
+  motionId: string;
+  isObjectionStake?: boolean;
+};
+
+const MSG = defineMessages({
+  motionTitle: {
+    id: 'dashboard.ActionsPage.TotalStakeWidget.motionTitle',
+    defaultMessage: 'Stake',
+  },
+  objectionTitle: {
+    id: 'dashboard.ActionsPage.TotalStakeWidget.objectionTitle',
+    defaultMessage: 'Goal',
+  },
+  stakeProgress: {
+    id: 'dashboard.ActionsPage.TotalStakeWidget.stakeProgress',
+    defaultMessage: '{totalPercentage}% of {requiredStake} {tokenSymbol}',
+  },
+  userStake: {
+    id: 'dashboard.ActionsPage.TotalStakeWidget.userStake',
+    defaultMessage: `You staked {userPercentage}% of this motion ({userStake} {tokenSymbol}).`,
+  },
+});
+
+const TotalStakeWidget = ({
```

Comment:
Why is this not integrated with the Motion Page ?

Reply:
Thought the issue was just doing the component and the saga associated with it, my bad